net-misc/feather/feather-1.0.1.ebuild

```
@@ -0,0 +1,107 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

We can use EAPI-8 now.

```suggestion
EAPI=8
```

---

Fixed.